aspnetcore/blazor/hosting-models.md

```diff
@@ -150,6 +80,21 @@ Blazor WebAssembly includes support for trimming unused code from .NET Core fram
 
 ::: moniker-end
 
+## Hosting model selection
+
+Choice of the Blazor hosting model is an early consideration for Blazor app development. The following table shows the primary considerations for selecting the hosting model.
+
+| &nbsp; | Blazor Server | Blazor WebAssembly |
+| --- | :---: | :---: |
+| Wide .NET Core API compatibility              | ✔️ | ❌ |
```

Comment:
I don't want this to become misleading. If we call it 'Wide' it feels like it's incomplete - it's just most of them.
Instead, if we call it "Complete" the perception will be more than Wide. So, should this be renamed to `Complete .NET Core API compatibility`?

Reply:
Sure, I was just going off of some Dan lingo, probably from a blog post. I'll change it to "Complete" on the next commit.

Reply:
@danroth27 I hope you have no objections regarding this.